ethereum/.env.template

```diff
@@ -24,3 +24,6 @@ PYTHNET_EMITTER= # 0xa27839d641b07743c0cb5f68c51f8cd31d2c0762bec00dc6fcd25433ef1
 # valid, after this duration, the price feed is stale and will be invalid.
 # This value should derive from Pyth to wormhole latency, and target chain blocktime and latency.
 VALID_TIME_PERIOD_SECONDS= # 60
+
+GOVERNANCE_CHAIN_ID= # 0x1
```

Comment:
why are these commented out?

Reply:
This one is not used and I forgot to uncomment it.

Reply:
This is the template file as a description.